docs/other/install-gwms-frontend.md

```diff
@@ -439,13 +439,152 @@ After configuring condor, be sure to restart condor:
+
+    :::console
+    root@host # voms-proxy-info --file %RED%<PATH_TO_NEW_PROXY.PROXY>%ENDCOLOR% -all
+    subject   : /DC=org/DC=opensciencegrid/O=Open Science Grid/OU=Services/CN=pilot/glidein2.chtc.wisc.edu/CN=proxy
+    issuer    : /DC=org/DC=opensciencegrid/O=Open Science Grid/OU=Services/CN=pilot/glidein2.chtc.wisc.edu
+    identity  : /DC=org/DC=opensciencegrid/O=Open Science Grid/OU=Services/CN=pilot/glidein2.chtc.wisc.edu
+    type      : proxy
+    strength  : 1024 bits
+    path      : %RED%PATH_TO_NEW_PROXY.PROXY%ENDCOLOR%
+    timeleft  : 23:30:51
+    key usage : Digital Signature, Key Encipherment, Data Encipherment
+    === VO GLOW extension information ===
+    VO        : GLOW
+    subject   : /DC=org/DC=opensciencegrid/O=Open Science Grid/OU=Services/CN=pilot/glidein2.chtc.wisc.edu
+    issuer    : /DC=org/DC=opensciencegrid/O=Open Science Grid/OU=Services/CN=glow-voms.cs.wisc.edu
+    attribute : /GLOW/Role=htpc/Capability=NULL
+    timeleft  : 23:30:51
+    uri       : glow-voms.cs.wisc.edu:15001
+
+#### Creting a script fro automatic proxy renewal
+
+This section describes how to create a script to automaticly renew proxies using `voms-proxy-fake`. More information on how to generate a proxy using `voms-proxy-fake` can be found in [section](#generating-a-proxy-using-voms-proxy-fake).
+
+1. Create a script that looks like to generate proxies called `/etc/cron.hourly/renew_proxies.sh`
+
+        :::file
+        #!/bin/sh                                                                                                                          
+        # renew frontend proxy                                                                                                             
+        grid-proxy-init  -debug                      \
+         -cert %GREEN%<VO_FRONTEND_CERT>%ENDCOLOR%   \
+         -key  %GREEN%<VO_FRONTEND_KEY> %ENDCOLOR%   \
+         -out %GREEN%<VO_FRONTEND_PROXY>%ENDCOLOR%   \
+         -hours 24
+        chown frontend:frontend	%GREEN%<VO_FRONTEND_PROXY>%ENDCOLOR%
+
+        # renew pilot proxy                                                     
+        voms-proxy-fake  -debug                         \
+         -cert %RED%<PILOT_CERT>%ENDCOLOR%              \
+         -key %RED%<PILOT_KEY>%ENDCOLOR%                \
+         -out %RED%<PILOT_PROXY>%ENDCOLOR%              \
+         -fqan /GLOW/Role=htpc/Capability=NULL          \
+         -voms GLOW                                     \
+         -hours 24                                      \
+         -hostcert /etc/grid-security/voms/vomscert.pem \
+         -hostkey /etc/grid-security/voms/vomskey.pem   \
+         -uri glow-voms.cs.wisc.edu:15001
+        chown frontend:frontend %RED%<PILOT_PROXY>%ENDCOLOR%
+        
+
+2. Make the script executable
+
+        :::console
+        root@host # chmod +x /etc/cron.hourly/renew_proxies.sh
+
+3. Test the script
+
+        :::console
+        root@host # run-parts /etc/cron.hourly
```

Comment:
Wouldn't this run all scripts in `/etc/cron.hourly`? I think we should just have the run `renew_proxies.sh` by hand

Reply:
Yea but in some cases it is better to test how it would run. But ok I changed it